README.md

```diff
@@ -174,14 +176,16 @@ for Intel IPU ES2K.
    # git checkout ipdk_v23.07
    ```
 
-3. Build the P4 Kubernetes binaries:
-  ```bash
-  # make build
-  ```
-  Then build the Kubernetes container images:
-  ```bash
-  # make docker-build
-  ```
+3. Build the P4 Kubernetes binaries. Note that, by default, Makefile is
```

Comment:
Though the default option is es2k, I believe we should mention how to build for both targets tagname=es2k, tagname=dpdk

Reply:
Yes, it has been updated with this detail about the tagname required for DPDK